autogpt_platform/backend/backend/blocks/orchestrator.py

```
@@ -439,7 +439,7 @@ class Input(BlockSchemaInput):
             description="The language model to use for answering the prompt.",
             advanced=False,
         )
-        credentials: llm.AICredentials = llm.AICredentialsField()
+        credentials: Optional[llm.AICredentials] = llm.AICredentialsField()
```

Ruff (0.16.1)

[error] Undefined name `Optional`

(F821)